packages/algorand-js/src/lib/constants.ts

```diff
@@ -4,3 +4,4 @@ export const MIN_UINT64 = BigInt("0");
 export const MAX_UINT8 = 255;
 export const MIN_UINT8 = 0;
 export const DEFAULT_STACK_ELEM = BigInt("0");
+export const MAX_STRING_SIZE = 4096;
```

Comment:
`MAX_STRING_SIZE` is ambigous. Maybe change to `MAX_CONCAT_STRING`?

Reply:
yes